contrib/cmake/preferred-ports-latest.txt

```diff
@@ -0,0 +1,44 @@
+asio/latest
+bzip2/latest
+c-ares/latest
+cryptopp/latest
+cryptopp-staticcrt/8.2.0
```

Comment:
why not latest?

Reply:
That one doesn't actually exist in vcpkg - it's a trick to allow us to build a nonstatic and a static version in the same vcpkg set (MEGAsync needs the static one for the updater, and nonstatic DLL for MEGAsync itself)